biz.aQute.bndlib/src/aQute/bnd/build/ProjectBuilder.java

```diff
@@ -247,235 +253,6 @@ public Jar getLastRevision() throws Exception {
 	}
 
 	/**
-	 * This method attempts to find the baseline jar for the current project. It
-	 * reads the -baseline property and treats it as instructions. These
-	 * instructions are matched against the bsns of the jars (think sub
-	 * builders!). If they match, the sub builder is selected.
-	 * <p>
-	 * The instruction can then specify the following options:
-	 * 
-	 * <pre>
-	 * 	version : baseline version from repository
-	 * 	file    : a file path
-	 * </pre>
-	 * 
-	 * If neither is specified, the current version is used to find the highest
-	 * version (without qualifier) that is below the current version. If a
-	 * version is specified, we take the highest version with the same base
-	 * version.
-	 * <p>
-	 * Since baselining is expensive and easily generates errors you must enable
-	 * it. The easiest solution is to {@code -baseline: *}. This will match all
-	 * sub builders and will calculate the version.
-	 * 
-	 * @return a Jar or null
-	 */
-	public Jar getBaselineJar() throws Exception {
```

Comment:
I'm confused! the ProjectBuilder does indeed call down to the BaselineProcessor.